src/templates/globals/wrapper/index.tsx

```diff
@@ -14,6 +14,7 @@ import { NodeBanner } from '@/types/dataTypes/drupal/node'
 import { BannerDisplayType, BannerTypeMapping } from '@/data/queries/banners'
 import { Header } from '../header'
 import { Footer } from '../footer/index'
+import { handleSkipLink } from '../../../lib/utils/handleSkipLink'
```

Comment:
**CONSIDER**
Now that this is moved to the general `utils` directory, I think we could consider an absolute path using `@`. I don't think it's necessary (or advised) for the `Header` and `Footer` imports above it, but it seems to make sense here.

Reply:
Good call, fixed